.gitignore

```diff
@@ -3,4 +3,4 @@
 /build/
 /cmd/version.pony
 /_corral/
-/_repos/
+/_repos/
```

Comment:
i think this change can be reverted.

Reply:
Why? I don't think this directory should be in git.

Reply:
Sorry, what? You removed a new line. That's the only change here. 

Are you saying git ignore shouldn't be checked in? I would disagree with that.

Reply:
Oh, I misread this change, I thought that the `_repos` directory had been removed from `.gitignore` file and I thought I added it back.  I'll revert the change.